tcms/testruns/static/testruns/js/get.js

```diff
@@ -180,13 +180,36 @@ function renderTestExecutions(testExecutions) {
     })
 
     treeViewBind();
+    $('.test-executions-count').html(testExecutions.length)
     renderAdditionalInformation(testExecutions, testCaseIds)
 }
 
 function renderAdditionalInformation(testExecutions, testExecutionCaseIds) {
-    $('.test-executions-count').html(testExecutions.length);
 
     renderTestCaseInformation(testExecutions, testExecutionCaseIds)
+
+    testExecutions.forEach(testExecution => {
+        jsonRPC('TestExecution.history', testExecution.id, history => {
+            const liHistory = $(`.test-execution-${testExecution.id} .history-container`)
+            history.forEach(h => {
+                liHistory.append(renderHistoryEntry(h))
+            })
+        })
+    })
+}
+
+function renderHistoryEntry(historyEntry) {
+    if (!historyEntry.history_change_reason) {
+        return ''
+    }
```

Comment:
Is this to handle the None value ? If yes then remove it b/c `None` is at least visible but empty string isn't. If there's no history beyond a certain point there isn't something we can do about that.

Reply:
```
> jsonRPC('TestExecution.history', 1, console.log)
(4) [{…}, {…}, {…}, {…}]
0: {history_user__username: "atodorov", history_change_reason: "--- close_date↵+++ close_date↵@@ -1 +1 @@↵-2018-11…421↵--- status_id↵+++ status_id↵@@ -1 +1 @@↵-3↵+8", history_date: "2019-02-11T19:06:53.515"}
1: {history_user__username: "atodorov", history_change_reason: "--- close_date↵+++ close_date↵@@ -1 +1 @@↵-2018-11…tatus_id↵+++ case_run_status_id↵@@ -1 +1 @@↵-3↵+4", history_date: "2019-01-14T13:02:25.267"}
2: {history_user__username: "atodorov", history_change_reason: "--- tested_by_id↵+++ tested_by_id↵@@ -1 +1 @@↵-Non…tatus_id↵+++ case_run_status_id↵@@ -1 +1 @@↵-1↵+3", history_date: "2018-11-27T21:06:48.067"}
3: {history_user__username: "atodorov", history_change_reason: null, history_date: "2018-11-27T21:04:34.859"}
```
This is to make sure, we don't render an empty entry for the last row that has no change_reason